### PR TITLE
check that swap exist before removing it

### PR DESCRIPTION
**Pull request by ptqa**
This fixes issue #20.

### CODE REVIEW

File: libraries/swapfile_provider.rb [hunk 1]
```diff
@@ -89,7 +89,7 @@ def swapoff
       end
 
       def remove_swapfile
-        ::FileUtils.rm(@new_resource.path)
+        ::FileUtils.rm_f(@new_resource.path)
```
Comment (sethvargo) [inline libraries/swapfile_provider.rb]: I think we should leave this as-is. If this function is called and the swap does not exist, I think it should fail. We should guard against calling this function higher up, as you did above. Thoughts?

Comment (ptqa) [inline libraries/swapfile_provider.rb]: If we removed it on previous chef run, why we should fail on next one?

**Comment by ptqa** [comment]
@sethvargo please check updated commit

**Comment by sethvargo** [comment]
Thanks!

**Pull request merged**